### PR TITLE
Upgrade snippets for v7 TS changes

### PR DESCRIPTION
**Pull request by kasperpeulen**
## What I did

- Upgrade snippets for v7 TS changes

### CODE REVIEW

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@kasperpeulen, thanks for the fantastic job on this one. From my end, just a couple of small tweaks and this is good to go. Let me know once they're addressed.

File: docs/snippets/angular/button-group-story.ts.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ```ts
 // ButtonGroup.stories.ts
 
-import { Meta, moduleMetadata, StoryFn } from '@storybook/angular';
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply docs/snippets/angular/button-group-story.ts.mdx:5]: Done that

File: docs/snippets/angular/storybook-preview-global-decorator.ts.mdx [hunk 2]
```diff
@@ -1,9 +1,9 @@
-```js
+```ts
 // .storybook/preview.js
```
[the remark this answers is not in the document]
Comment (kasperpeulen) [reply docs/snippets/angular/storybook-preview-global-decorator.ts.mdx:2]: Yes already available! Changed the filename.

**Review by jonniebigodes — APPROVED** [review]
All good on my end! This is good to merge! Thanks for taking care of the feedback so promptly @kasperpeulen

**Pull request merged**